eli5/formatters/image.py

```diff
@@ -0,0 +1,326 @@
+# -*- coding: utf-8 -*-
+from typing import Union, Optional, Callable
+
+import numpy as np # type: ignore
+from PIL import Image # type: ignore
+import matplotlib.cm # type: ignore
+
+from eli5.base import Explanation
+
+
+# FIXME: in autodoc the interpolation and colormap defaults show
+# something like: 
+# interpolation=<Mock id='139885759460352'>, colormap=<Mock id='139885771653072'>
+# ID's instead of actual names.
+def format_as_image(expl, # type: Explanation
+    interpolation=Image.LANCZOS, # type: int
+    colormap=matplotlib.cm.magma, # type: Callable[[np.ndarray], np.ndarray]
+    alpha_limit=0.65, # type: Optional[Union[float, int]]
+    ):
+    # type: (...) -> Image
+    """ Format a :class:`eli5.base.Explanation` object as an image.
+
+    Note that this formatter requires ``matplotlib`` and ``Pillow`` optional dependencies.
+    
+    Parameters
+    ----------
+    interpolation: int, optional
+        Interpolation ID or Pillow filter to use when resizing the image.
+        
+        Example filters from PIL.Image
+            * ``NEAREST``
+            * ``BOX``
+            * ``BILINEAR``
+            * ``HAMMING``
+            * ``BICUBIC``
+            * ``LANCZOS``
+
+        See also https://pillow.readthedocs.io/en/stable/handbook/concepts.html#filters.
+
+        *Note that these attributes are integer values*.
+
+        Default is ``PIL.Image.LANCZOS``. 
+
+    colormap: callable, optional
+        Colormap scheme to be applied when converting the heatmap from grayscale to RGB.
+        Either a colormap from matplotlib.cm, 
+        or a callable that takes a rank 2 array and 
+        returns the colored heatmap as a [0, 1] RGBA numpy array.
+
+        Example colormaps from matplotlib.cm
+            * ``viridis``
+            * ``jet``
+            * ``binary``
+        
+        See also https://matplotlib.org/3.1.0/gallery/color/colormap_reference.html.
```

Comment:
It's possible to link to the latest version:
```suggestion
        See also https://matplotlib.org/gallery/color/colormap_reference.html.
```